src/ray/raylet/scheduling/cluster_resource_data.h

```diff
@@ -237,3 +239,7 @@ NodeResources ResourceMapToNodeResources(
 TaskRequest ResourceMapToTaskRequest(
     StringIdMap &string_to_int_map,
     const std::unordered_map<std::string, double> &resource_map);
+
```

Comment:
Can you move this to `cluster_resource_data` (and ideally make the logic there call this function?)